utils/json_profiles_merger.py

```diff
@@ -43,6 +48,10 @@ def main(argv):
   # Discard the first argument (the binary).
   input_profiles = argv[1:]
 
+  if FLAGS.input_profile_dir:
+    # Add any globbed files from the input_dir to the list.
+    input_profiles += glob(FLAGS.input_profile_dir + "/*.json*")
```

Comment:
How do you invoke the profile merger?

If it's from a shell I don't see why this code modification is better than just doing the globbing on the shell?

Reply:
An example from yesterday when I implemented this.

Globbing on the command line for large (1400~ files), leads to:
```
FATAL: execv of '/bin/bash' failed: (error: 7): Argument list too long
```

Reply:
I see :-)

Your glob expression wonders me: Bazel by default writes the profile to a file called `command.profile.gz` and in the benchmark they also end with `profile.gz`: https://github.com/bazelbuild/bazel-bench/blob/master/benchmark.py#L208

Reply:
Agreed, the glob isn't perfect. Any suggestions are welcome :)

Reply:
Would `glob(FLAGS.input_profile_dir + "/*.profile.gz")` work for you?

Reply:
Seems reasonable, I can always preprocess my file names.